### DISCUSSION TITLE
Color font in splash layout

### DISCUSSION THREAD

**Discussion by edualvarado** [mmistakes/minimal-mistakes#2436] [Q&A] [answered]
Hello! I am using a splash layout for my home-page stored with GitHub. I know how to change the font color, but I cannot find where to change the color of the text that appears in front of the "splash" background. For me, it always appears white.

For example, in this splash page: [https://mmistakes.github.io/minimal-mistakes/](url), how would it be to change the "_Minimal Mistakes, A flexible layout..._" text to another color and not only white?

Thanks!

**Comment by mmistakes** [comment, marked as answer]
Via CSS. Use your browser's web developer tools to inspect an element and determine the class selector(s) you need to modify.

![Screen Shot 2020-03-15 at 2 41 53 PM](https://user-images.githubusercontent.com/1376749/76708170-301fb500-66cb-11ea-910f-1ec35c8bc16e.png)

**Comment by edualvarado** [comment]
I did not know about this utility, I am quite a noob with html. It worked fine :) Thank you!
